trompace/queries/mediaobject.py

```diff
@@ -0,0 +1,54 @@
+# Generate GraphQL queries for queries pertaining to media objects.
+from trompace.exceptions import UnsupportedLanguageException, NotAMimeTypeException
+from trompace.queries.templates import format_query
+from trompace import StringConstant, _Neo4jDate, filter_none_args, docstring_interpolate
+from trompace.constants import SUPPORTED_LANGUAGES
+
+MEDIAOBJECT_ARGS_DOCS = """identifier: The identifier of the media object in the CE.
+        name: The name of the media object.
```

Comment:
this documentation now doesn't match up with the fields of the method that it's used on, can you make sure that the documentation makes sense compared to the method?

I used this specific system of having a separate variable for the arguments in the case of mutations where we need to share the documentation between the create and update mutation. In the case where we only have one method (like query_mediaobject), we can just leave the documentation in the docstring.